assets/src/scripts/hydrograph/index.spec.js

```diff
@@ -149,4 +179,4 @@ const MOCK_DATA = `[
         "time": "2018-01-03T18:15:00.000Z",
         "value": 24
     }
-]`
+];
```

Comment:
Hmmm... why did my code work... it shouldn't have? I must've originally intended for MOCK_DATA to be a json string to be parsed...

Reply:
I think because the test only tests to see if the path is there, the path is still generated even if the time series is empty.